fastplotlib/graphics/line.py

```diff
@@ -8,6 +8,11 @@
 
 
 class LineGraphic(Graphic, Interaction):
+    feature_events = [
+        "data",
+        "colors",
+        "cmap",
```

Comment:
there is technically no `"cmap"` event, the CmapFeature is just a wrapper around ColorsFeature (i.e. it subclasses ColorFeature) and just parses cmap strings into an n_points, RGBA colors array based on the user-provided indexing.